frontend/packages/citizen-frontend/src/localization/en.ts

```diff
@@ -124,6 +124,40 @@ const en: Translations = {
       },
       returnToPreviousPage: 'Return'
     }
+  },
+  applicationsList: {
+    title:
+      'Applying for early childhood education or a club and enrolling for pre-primary education',
+    summary:
+      'A child’s guardian can submit an application for early childhood education or a club and enrol the child to pre-primary education. Information on the guardian’s children is automatically retrieved from the Digital and Population Data Services Agency and displayed in this view.',
+    pageLoadError: 'Failed to load guardian applications',
+    type: {
+      daycare: 'Daycare application',
+      preschool: 'Early education application',
+      club: 'Club application'
+    },
+    unit: 'Yksikkö',
+    period: 'Ajalle',
+    created: 'Luotu',
+    modified: 'Muokattu',
+    status: {
+      title: 'Tila',
+      CREATED: 'Luonnos',
+      SENT: 'Lähetetty',
+      WAITING_PLACEMENT: 'Käsiteltävänä',
+      WAITING_DECISION: 'Käsiteltävänä',
+      WAITING_UNIT_CONFIRMATION: 'Käsiteltävänä',
+      WAITING_MAILING: 'Käsiteltävänä',
+      WAITING_CONFIRMATION: 'Vahvistettavana huoltajalla',
+      REJECTED: 'Paikka hylätty',
+      ACTIVE: 'Paikka vastaanotettu',
+      CANCELLED: 'Poistettu käsittelystä'
+    },
+    openApplicationLink: 'Näytä hakemus',
+    confirmationLinkInstructions:
+      'Päätökset-välilehdellä voit lukea päätöksen ja hyväksyä/hylätä tarjotun paikan',
+    confirmationLink: 'Siirry vahvistamaan',
+    newApplicationLink: 'Uusi hakemus'
```

Comment:
At least these already have localizations in `enduser-frontend`. Same with Swedish localizations.